File: June-18/10-06-18/c++/01-Jump-Out.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    cin >> n;
    int ans = 0;
    for(int i = 0; i < n; i++)
    {
        int tmp;
        cin >> tmp;
        if(ans == 0 and n-tmp - i <= 0)
            ans = i + 1;
    }
    cout << ans << endl;
}

File: June-18/10-06-18/java/01-Jump-Out.java

import java.util.*;


class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int ans = 0;
        for(int i = 0; i < n; i++)
        {
            int tmp = input.nextInt();
            if(ans == 0 && n - tmp - i <= 0)
                ans = i + 1;
        }
        System.out.println(ans);
        
    }
}
